### ISSUE TITLE
making a corpus of articles

### ISSUE THREAD

**Issue by Armand1** [Armand1/Women-in-the-BMJ#2]
The latest collection of **74937** articles is very clean.  But we still want to clean it a bit more

For example, there are still a relatively small number of duplicate titles.  These are always suspicious as they may be regular features. e.g., "website of the week".  I have identified all duplicate titles and vetted them, removing those that are probably not of direct clinical significance but are about NHS politics and the like.  This removes 579 articles. 

I have also checked the total word count of articles.  Some are very short (e.g., 1 or 2 words).  These seem to be due to errors in the dataset.  But, for safety, I have excluded all articles with fewer than 50 words.  This removes 364 articles. Even if the words are accurate, the LDA won't do a good job on very short articles.

We have a dataset of word counts.  The topic probabilities are based on these, but weirdly this dataset is slightly smaller than the topic probability dataset.  For safety I exclude all articles that are not in both.  This removes about 1000 articles.

In addition, examination of the word frequencies showed that there were really weird spikes around 1997.  Examination finally revealed a very strange pattern.  I counted the number of words in each article in each issue.  These are usually unique, for example, 515, 1723 etc.  But in 1997 I found a whole bunch of articles with identical word counts (e.g., 10 articles with word counts of 4736).  Something is clearly wrong --- presumably an error in how the articles were parsed.  Now I could just chuck the entire year, but I have instead removed all articles with duplicate word counts:  733 articles in all. The problem starts at 04/01/1997 and ends at 14/02/1998.  **Even though I have removed "duplicate" articles, if we see something funny in topic frequencies around 1997 it could be due to this**. 

So now we have **72494** articles. 

The topic probabilities for these papers (with their metadata) are in the dataset 
**BMJ_171119_topicprobabilities_edited.csv**
The word counts for these papers (with their metadata) are in the dataset
**BMJ_171119_wordsbypapers_edited.csv**

These should be the basis of our future studies

**Comment by evanhamulyak** [comment]
Do we have any idea what the 'articles' with less than 50 words could be? Even most abstract are longer than that... I think we can absolutely justify doing so, explicitly write it down and keep in mind indeed if see something around 1997.
The set of articles is still higher than the last time, if I am correct, last time you ran the analyses on a 67.000+ something set.

**Comment by Armand1** [comment]
Yes, the number of articles is larger.... little bit unclear why that is.  Here I am kinda at the mercy of whatever Austin gives me.  No, don't know what the <50 word articles are.  I think that that they may be OCR errors.  (Some have 1 word.)

**Comment by Armand1** [comment]
It's clear that paper length is having an effect on our results.  And that the OCR errors in 1997 are giving us really long papers.  I have therefore constructed new datasets excluding 04/01/1997:14/02/1998. These are called 

So now we have **71866** articles


The topic probabilities for these papers (with their metadata) are in the dataset
**BMJ_1011219_topicprobabilities_edited.csv**
The word counts for these papers (with their metadata) are in the dataset
**BMJ_011219_wordsbypapers_edited.csv**

These should be the basis for our studies
